shared/chat/conversation/input-area/normal/platform-input.native.js

```diff
@@ -29,6 +35,14 @@ class PlatformInput extends Component<PlatformInputProps, State> {
     this.props.inputSetRef(ref)
   }
 
+  _explodingPickerToggle = () => {
```

Comment:
same here, this and `state.explodingPickerOpen` can go away to use the props from the HOC